src/multiple-choice/components/runtime.tsx

```diff
@@ -117,6 +134,16 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
             readOnly={readOnly}
             disabled={readOnly}
           />
+          {choice.correct !== undefined && choice.correct === true && showCorrect &&
```

Comment:
```suggestion
          {choice.correct && showCorrect &&
```
The test against `undefined` is redundant (if it's `true` it can't be `undefined`). Furthermore, unless there's some need to distinguish truthy values other than `true`, the simple boolean test should be sufficient.

Reply:
Good call.  That was verbose.  Fixed.